### DISCUSSION TITLE
Notifiche Azionabili legate al dispositivo

### DISCUSSION THREAD

**Discussion by Matty1184** [indomus/forum#2367] [HUB: Home Assistant]
Ciao a tutti, vorrei lanciare uno script alla pressione di un pulsante (da plancia) + conferma di una notifica di lanciare lo script.
non riesco a trovare una soluzione però al fatto di mandare la notifica al dispositivo che ha premuto il pulsante.
esempio:
script che chiude tutte le serrande,
pulsante che invia una notifica "vuoi chiudere tutte le serrande" SOLO al dispositivo/utente che ha premuto il pulsante
se l'action del pulsante è OK avvia lo script.
le notifiche con le action se utilizzo il servizio notify arrivano a tutti i dispositivi, salvo non inserisca un dispositivo specifico ma non è il mio scopo.
possibili soluzioni?

**Comment by 50494554524F** [comment]
devi usare il campo "confirmation" di "tap_action"

**Reply by Matty1184** [reply]
Grazie mille, questo lo stavo già "sperimentando" ma vorrei che alla fine dello script inviare una notifica che comunica l'avvenuta esecuzione dello script. questa notifica però vorrei che venisse inviata solo all'utente che ha lanciato lo script.

**Reply by 50494554524F** [reply]
non era chiaro... allora... 

puoi recuperare lo user_id dell'utente che ha cliccato nativamente all'interno delle automazioni, quindi potesti scatenare un evento dallo script e far partire un'automazione... altrimenti devi usare un'integrazione che trovi in hacs (ora non mi torna in mente)

sennò puoi semplicemente mostrare un popup tramite "browser_mod" che trovi sempre in hacs (o sicuramente con altre)
